integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -817,14 +759,16 @@ private int getAdminSericeLBNodePort() throws Exception {
 
     logger.info("Cmd to get the admins service node port " + cmd);
 
-    ExecResult result = ExecCommand.exec(cmd.toString());
-    if (result.exitValue() == 0) {
-      return new Integer(result.stdout().trim()).intValue();
-    } else {
-      throw new RuntimeException("Cmd failed " + result.stderr() + " \n " + result.stdout());
-    }
+    ExecResult result = TestUtils.exec(cmd.toString());
+    return new Integer(result.stdout().trim()).intValue();
   }
 
+  /**
+   * Create a map with attributes required to create PV and create PV dir by calling
+   * PersistentVolume
+   *
+   * @throws Exception
```

Comment:
you should explain when/why the exception is thrown on the @throws line

Reply:
Done